### PR TITLE
PCHR-568: Appraisal Cycle - Appraisals in Cycle (static)

### PR DESCRIPTION
**Pull request by AkA84**
This PR adds the static version of the "Appraisals in Cycle" tab of the individual Appraisal page

appraisals-in-cycle

##### View Cycle modal
view-modal

##### Delete Cycle dialog
delete-modal

##### Send Notification/Reminder modal
reminder-modal

#### `.text-default`, `.text-default-darker`
This are simple helper classes that applies the `$gray-dark` and `$gray-darker` colors respectively to the content

#### `.dl-horizontal-half`
dl-horizontal-half

This is a variation of `.dl-horizontal` with an offset of half the width of the offset in the standard version

### CODE REVIEW

File: uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html [hunk 1]
```diff
@@ -62,202 +62,951 @@ <h2 class="panel-title pull-left panel--appraisal-cycle-header__title">
...
+                                                            <label for="self-due-from" class="control-label col-sm-7">
+                                                                Self Appraisal Due From:
+                                                            </label>
+                                                            <div class="col-sm-5">
+                                                                <div class="input-group input-group-sm input-group-unstyled">
+                                                                    <input
+                                                                        type="text"
+                                                                        class="form-control"
+                                                                        id="self-due-from"
+                                                                        datepicker-popup="dd/MM/yyyy"
```
Comment (kkalamarski) [inline uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html]: Can't you use DateFormat service to provide dateformat?

Comment (AkA84) [inline uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html]: Sure, but this task is just for creating a static version, the controllers i wrote were just for showing the modals

Comment (kkalamarski) [inline uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html]: Ah, okay, let it be. What's the purpose of static version?

Comment (AkA84) [inline uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html]: Creating the markup and the css to get a page that matches the original design. The focus here is HTML and CSS, not JS

Comment (kkalamarski) [inline uk.co.compucorp.civicrm.appraisals/views/appraisal-cycle.html]: Ok, understood.

**Comment by kkalamarski** [comment]
Approved

**Pull request merged**